pom.xml

```diff
@@ -59,7 +59,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR PROJECT_ID HERE</deploy.projectId>
+          <deploy.projectId>bksundar-step-2020</deploy.projectId>
```

Comment:
The project id for your capstone is step170-2020. Also remove TODO when you are done.

Reply:
Sure, will change that right away.